using-timescaledb/distributed-hypertables.md

```diff
@@ -0,0 +1,124 @@
+using distributed hypertables that might be good to review before getting
+started.
+
+### Creating Distributed Hypertables [](create)
+
+As with normal hypertables, start by creating a regular
+table to hold the data, then transform it into a distributed
+hypertable. In contrast to creating a normal hypertable, we use
+the function
+[`create_distributed_hypertable`][create_distributed_hypertable].
+
+```sql
+CREATE TABLE conditions (
+  time        TIMESTAMPTZ       NOT NULL,
+  location    TEXT              NOT NULL,
+  temperature DOUBLE PRECISION  NULL,
+  humidity    DOUBLE PRECISION  NULL
+);
+
+SELECT create_distributed_hypertable('conditions', 'time', 'location');
+```
+
+>:WARNING: If there are no data nodes available, you will get an error
+>when executing `create_distributed_hypertable` and the distributed
+>hypertable will not be created.
+
+This creates a multi-dimensional distributed hypertable
+partitioned along `time` and `location`.
+You can now insert data into the distributed hypertable and
+it will automatically be partitioned on the available data nodes
+by using the provided space partition. You
+can find more information for how to work with data in hypertables in the
+section [Creating Hypertables][creating-hypertables].
+
+### Changing the Number of Data Nodes for a Distributed Hypertable [](changing-data-nodes)
+
+By default, distributed hypertables will use all available data nodes,
+but individual data nodes can be added and removed from already
+created distributed hypertables using
+[`add_data_node`][add_data_node]/[`attach_data_node`][attach_data_node] and
+[`detach_data_node`][detach_data_node].
+
+For example, if you add a new data node to the distributed database,
+it will not automatically be
+added to existing distributed hypertables, so it is necessary to attach
+it explicitly.
+
+```sql
+SELECT add_data_node('node3', host => 'dn3.example.com');
+
+SELECT attach_data_node('node3', hypertable => 'conditions');
+```
+
+Once the data node has been attached, any new chunks will also be
+distributed to the hypertable on the new data node.
+
+In a similar way, if you want to remove a data node from a distributed
+hypertable, you can use [`detach_data_node`][detach_data_node].
+
+```sql
+SELECT detach_data_node('node1', hypertable => 'conditions');
+```
+
+Note that you cannot detach a data node that still holds data for the
+hypertable. However, if the distributed hypertable is subject to a
+retention policy, it is possible to block new data on a data node and
+wait until the existing data on the data node moves outside the
+retention window. This is achieved by first blocking new chunks on the
+data node:
+
+```sql
+SELECT block_new_chunks('node1', hypertable => 'conditions');
+```
+
+This will prohibit new data from being stored on the data node. Once
+the data node's data is outside the retention window,
+[`drop_chunks`][drop_chunks] can be used to delete the data on the data
+node.
```

Comment:
```suggestion
This will prevent data from being added or updated on the data node. Once
the data node's data is outside the retention window,
[`drop_chunks`][drop_chunks] can be used to delete the data on the data
node.
```